recipes/python/python3.yaml

```diff
@@ -90,12 +90,15 @@ multiPackage:
             rsync -a "${BOB_DEP_PATHS[python::python3-wheel]}/" install/
 
         packageScript: |
-            autotoolsPackageTgt
+            autotoolsPackageBin
+            autotoolsPackageDev
             ln -sf python3 usr/bin/python
 
         provideDeps: [ "*-tgt" ]
         provideTools:
-            python3: usr/bin
+            python3:
+                path: usr/bin
+                netAccess: True
```

Comment:
I think this is too coarse. Every recipe that uses the "python3" tool will get network access. I think this should be restricted to where it's really needed. Calling pip and downloading from the internet is probably restricted to particular recipes. Would it be feasible to enable `buildNetAccess` there instead of centrally here?

Reply:
Exactly.